forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -286,7 +290,52 @@ def form_design_update(data, form_id):
         formio_service = FormioService()
         form_io_token = formio_service.get_formio_access_token()
         response = formio_service.update_form(form_id, data, form_io_token)
-        FormHistoryService.create_form_log_with_clone(data=data)
+        FormProcessMapperService.execute.submit(
+            FormHistoryService.create_form_log_with_clone, data=data
+        )
+        return response
+
+    @staticmethod
+    def create_form(data, is_designer):
+        """Service to handle form create."""
+        # Initialize formio service and get formio token to create the form
+        formio_service = FormioService()
+        form_io_token = formio_service.get_formio_access_token()
+        # creating form and get response from formio
+        response = formio_service.create_form(data, form_io_token)
+        form_id = response.get("_id")
+        # create default data form mapper table
+        mapper_data = {
+            "formId": form_id,
+            "formName": response.get("title"),
+            "description": data.get("description", ""),
+            "formType": response.get("type"),
+            "processKey": response.get("name"),
+            "processName": response.get("name"),
```

Comment:
is it correct that we are setting processKey and processName from 'name' ? @shuhaib-aot

Reply:
form name is used as process_name & process_key